app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: "Fees and financial support for non-UK trainee teachers"
+subcategory: Train to teach in England as a non-UK citizen
```

Comment:
@laurahermionetennant92 I don't think the sub category title should be the same as the title of one of the cards. So I think it could be good to make it less similar and a little more broad - maybe something along the lines of 9but not this) if you want to train to teach.

We also have non-UK in the page title so I don't think we need to include it in the sub head for this or the teacher one.

Reply:
Makes sense